docs/pages/server-access/guides/azure-discovery.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Configure Teleport to Automatically Enroll Azure Virtual Machines (Preview)
+description: How to configure Teleport to automatically enroll Azure Virtual Machines.
```

Comment:
```suggestion
description: How to configure Teleport to automatically enroll Azure virtual machines.
```
If Azure Virtual Machines is the official product name, though, we should capitalize it everywhere